test/behavior/bugs/11995.zig

```diff
@@ -0,0 +1,24 @@
+const builtin = @import("builtin");
+
+fn wuffs_base__make_io_buffer(arg_data: wuffs_base__slice_u8, arg_meta: wuffs_base__io_buffer_meta) callconv(.C) void {
+    _ = arg_data;
+    _ = arg_meta;
+}
+const wuffs_base__io_buffer_meta = extern struct {
+    wi: usize,
+    ri: usize,
+    pos: u64,
+    closed: bool,
+};
+const wuffs_base__slice_u8 = extern struct {
+    ptr: [*c]u8,
+    len: usize,
+};
+test {
+    if (builtin.zig_backend == .stage2_wasm) return error.SkipZigTest; // TODO
+    if (builtin.zig_backend == .stage2_x86_64) return error.SkipZigTest; // TODO
+    if (builtin.zig_backend == .stage2_x86) return error.SkipZigTest; // TODO
+    if (builtin.zig_backend == .stage2_aarch64) return error.SkipZigTest; // TODO
+    if (builtin.zig_backend == .stage2_arm) return error.SkipZigTest; // TODO
+    _ = wuffs_base__make_io_buffer(undefined, undefined);
```

Comment:
need to actually test that the code runs correctly; it's not enough to just check that the compiler doesn't crash.

Reply:
Alright, I tried to come up with some things I could do there.